velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -155,5 +155,39 @@ TEST_F(ProbabilityTest, binomialCDF) {
       binomialCDF(-2, 0.5, -1), "numberOfTrials must be greater than 0");
 }
 
+TEST_F(ProbabilityTest, weibullCDF) {
+  const auto weibullCDF = [&](std::optional<double> a,
+                              std::optional<double> b,
+                              std::optional<double> value) {
+    return evaluateOnce<double>("weibull_cdf(c0, c1, c2)", a, b, value);
+  };
+
+  EXPECT_EQ(weibullCDF(1.0, 1.0, 0.0), 0.0);
+  EXPECT_EQ(weibullCDF(1.0, 1.0, 40.0), 1.0);
+  EXPECT_EQ(weibullCDF(1.0, 0.6, 3.0), 0.99326205300091452);
+  EXPECT_EQ(weibullCDF(1.0, 0.9, 2.0), 0.89163197677810413);
+
+  EXPECT_EQ(weibullCDF(std::nullopt, 1.0, 0.3), std::nullopt);
+
+  EXPECT_EQ(weibullCDF(1.0, std::nullopt, 0.2), std::nullopt);
+
+  EXPECT_EQ(weibullCDF(1.0, 0.4, std::nullopt), std::nullopt);
+
+  EXPECT_EQ(weibullCDF(kDoubleMin, 1.0, 2.0), 0.63212055882855767);
```

Comment:
Just curious : Why are we not testing varied combinations for value parameter ( < 0, kDoubleMin, kDoubleMax, kInf, kNan) etc ?

Reply:
Added and tested.